src/Resources/views/CRUD/base_acl_macro.html.twig

```diff
@@ -59,5 +56,5 @@ file that was distributed with this source code.
         <div class="well well-small form-actions">
             <input class="btn btn-primary" type="submit" name="btn_create_and_edit" value="{{ 'btn_update_acl'|trans({}, 'SonataAdminBundle') }}">
```

Comment:
Should not we also try to replace this input?

Reply:
Yes, but this could require more refactoring, e.g. we must check if the PHP form contains a button or not. Otherwise we could render the button twice or never.